feature-utils/poly-look/src/css/themes.css

```diff
@@ -21,15 +21,37 @@
 .poly-theme-light {
   --theme-primary-color: var(--color-gray-50);
   --theme-secondary-color: var(--color-indigo-950);
+  --theme-tertiary-color: var(--color-red-300);
   --theme-small-print-color: var(--color-gray-500);
-  --theme-info-button-color: var(--color-gray-300);
```

Comment:
this one is used by the icon button and it always has the same color, I think it's safe to move it under `poly-theme`